cmd/kpod/load.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"io"
+	"os"
+
+	"io/ioutil"
+
+	"github.com/containers/storage"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+const (
+	dockerArchive = "docker-archive:"
+	oci           = "oci:"
+)
+
+type loadOptions struct {
+	input string
+	quiet bool
+}
+
+var (
+	loadFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "input, i",
+			Usage: "Read from archive file, default is STDIN",
+			Value: "/dev/stdin",
+		},
+		cli.BoolFlag{
+			Name:  "quiet, q",
+			Usage: "Suppress the output",
+		},
+	}
+	loadDescription = "Loads the image from docker-archive by default or from oci if the oci flag is set."
+	loadCommand     = cli.Command{
+		Name:        "load",
+		Usage:       "load an image from docker archive or oci",
+		Description: loadDescription,
+		Flags:       loadFlags,
+		Action:      loadCmd,
+		ArgsUsage:   "",
+	}
+)
+
+// loadCmd gets the image/file to be loaded from the command line
+// and calls loadImage to load the image to containers-storage
+func loadCmd(c *cli.Context) error {
+	store, err := getStore(c)
+	if err != nil {
+		return err
+	}
+
+	args := c.Args()
+	if len(args) > 0 {
+		return errors.New("too many arguments. Requires exactly 1")
+	}
+
+	input := c.String("input")
+	quiet := c.Bool("quiet")
+
+	if input == "/dev/stdin" {
+		fi, err := os.Stdin.Stat()
+		if err != nil {
+			return err
+		}
+		if !fi.Mode().IsRegular() {
+			outFile, err := ioutil.TempFile("/tmp", "kpod")
```

Comment:
I think we should just switch to /var/tmp for this. /var/tmp/ is used for temporary files which can be large.